payments/utils.py

```diff
@@ -6,6 +6,7 @@
 
 codesy_pct = Decimal('0.025')
 stripe_pct = Decimal('0.029')
+stripe_transfer = Decimal('0.005')
 stripe_transaction = Decimal('0.30')
```

Comment:
Since all these values are decimals, might help to comment that this is a fixed cost - not a percentage like the others.